src/main/java/uk/gov/companieshouse/bankruptofficersearch/api/service/impl/ScottishBankruptOfficerSearchServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.companieshouse.bankruptofficersearch.api.service.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.bankruptofficersearch.api.model.response.ScottishBankruptOfficerDetailsEntity;
+import uk.gov.companieshouse.bankruptofficersearch.api.model.response.ScottishBankruptOfficerSearchEntity;
+import uk.gov.companieshouse.bankruptofficersearch.api.model.response.ScottishBankruptOfficerSearchResultsEntity;
+import uk.gov.companieshouse.bankruptofficersearch.api.model.rest.ScottishBankruptOfficerDetails;
+import uk.gov.companieshouse.bankruptofficersearch.api.model.rest.ScottishBankruptOfficerSearch;
+import uk.gov.companieshouse.bankruptofficersearch.api.model.rest.ScottishBankruptOfficerSearchResults;
+import uk.gov.companieshouse.bankruptofficersearch.api.request.OracleQueryDaoImpl;
+import uk.gov.companieshouse.bankruptofficersearch.api.service.BankruptOfficerSearchService;
+import uk.gov.companieshouse.bankruptofficersearch.api.transformer.ScottishBankruptOfficerTransformer;
+
+@Service
+public class ScottishBankruptOfficerSearchServiceImpl implements BankruptOfficerSearchService {
+
+	@Autowired
+	private ScottishBankruptOfficerTransformer scottishBankruptOfficerTransformer;
+
+	@Autowired
+	private OracleQueryDaoImpl oracleQueryDao;
+
+	@Override
+	public ScottishBankruptOfficerSearchResults searchScottishBankruptOfficers(ScottishBankruptOfficerSearch search) {
+
+	    ScottishBankruptOfficerSearchEntity searchEntity = scottishBankruptOfficerTransformer.convertToSearchEntity(search);
+		ScottishBankruptOfficerSearchResultsEntity details = oracleQueryDao.getScottishBankruptOfficers(searchEntity);
+
+		return scottishBankruptOfficerTransformer.convertToSearchResults(details);
+	}
```

Comment:
Can the inconsistent formatting and use of annotations (`@Override`) be fixed please?

Reply:
Code has been reformatted and `@Override` has been added